rtp/RtcpHeader.h

```diff
@@ -29,6 +29,8 @@ struct RtcpHeader
     uint8_t packetType : 8;
     nwuint16_t length;
 
+    static constexpr size_t maxReportsBlocks() { return 31; }
```

Comment:
What was the reason to do these static const methods instead of a public static const member that can read like a property? I think you mentioned it that it was related to template use.

Reply:
On c++14 inline static variables are supported. They worked only if you use them inside of the scope where they are declared. But if you try to use it outside the linker will fail because by the linkage entry does not exists.

We have 2 option or use a static variable but assign the value on cpp file (but it will not be a constexpr anylonger) or use a constexpr function